sdk/src/main/resources/sidechain-sdk-settings.conf

```diff
@@ -19,6 +19,6 @@ sparkz {
     bindAddress = "127.0.0.1:9084"
     knownPeers = []
     agentName = "2-Hop"
-    maxPacketSize = 5242980
+    maxModifiersSpecMessageSize = 5242980
```

Comment:
I think, it must be 7mb as for EVM.
to be discussed

Reply:
Ok, set the fallback conf to 7MB